packages/dx-react-chart/demos/area/spline-area.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Spline Area
+
+The demo shows how to customize an [Area Series](../../docs/reference/area-series.md) to display a Spline Area series.
```

Comment:
Customize a series to display a series?

Reply:
Yes, in our widgets user can customize existing series type and get his series type

Reply:
Ok